action.yml

```
@@ -285,5 +285,5 @@ runs:
         installer -pkg swift-${{ inputs.tag }}-osx.pkg -target CurrentUserHomeDirectory
         rm -f swift-${{ inputs.tag }}-osx.pkg
 
-        echo "TOOLCHAINS=$(plutil -extract 'CFBundleIdentifier' xml1 ${HOME}/Library/Developer/Toolchains/swift-${{ inputs.tag }}.xctoolchain/Info.plist | xmllint --xpath '//plist/string/text()' -)" >> $GITHUB_ENV
+        echo "TOOLCHAINS=$(plutil -convert swift ${HOME}/Library/Developer/Toolchains/swift-${{ inputs.tag }}.xctoolchain/Info.plist -o - | grep '"org.swift.' | sed -E -e 's/.*"(org.swift[^"]*)".*/\1/')" >> $GITHUB_ENV
```

```suggestion
        echo "TOOLCHAINS=$(plutil -convert swift ${HOME}/Library/Developer/Toolchains/swift-${{ inputs.tag }}.xctoolchain/Info.plist -o - | sed -E -e '/"org.swift./s/.*"(org.swift[^"]*)".*/\1/')" >> $GITHUB_ENV
```

I think that we can get away with just a single sed expression.